src/containers/createEvent/ducks/actions.ts

```diff
@@ -0,0 +1,16 @@
+import { genericAsyncActions } from '../../../utils/asyncRequest';
+import { NewEventInformation } from './types';
+
+export const create = genericAsyncActions<NewEventInformation, any>();
+export const edit = genericAsyncActions<NewEventInformation, any>();
+export const deleteEvent = genericAsyncActions<number, any>();
```

Comment:
We could be consistent here with the naming. In other words, either have createEvent, editEvent, deleteEvent or create, edit, and delete.

The former is clearer to me

Reply:
Some weird keyword/reuse stuff but fixed it, kinda